src/common.ts

```diff
@@ -1,18 +1,9 @@
-export interface TransactionOptions {
-  readOnly?: boolean;
-  timeout?: number;
-  exactStaleness?: number;
-  readTimestamp?: Date;
-  returnTimestamp?: boolean;
-  strong?: boolean;
-}
-export interface CreateSessionOptions {
-  name: string;
-  labels: {[key: string]: string};
-  createTime: GetTimestamp;
-  approximateLastUseTime: GetTimestamp;
-}
-export interface GetTimestamp {
-  nanos: number;
-  seconds: number;
-}
+import {google as database_admin_client} from '../proto/spanner_database_admin';
+
+export type Schema =
+  | string
+  | string[]
+  | database_admin_client.spanner.admin.database.v1.IUpdateDatabaseDdlRequest
+  | {statements: string[]; operationId?: string};
```

Comment:
isn't `database_admin_client.spanner.admin.database.v1.IUpdateDatabaseDdlRequest` is an object that contains `statements` and `operationId`? Do you need another definition for that?

Reply:
my bad, don't required `statements` and `operationId`